### ISSUE TITLE
zlib.h: no such file

### ISSUE THREAD

**Issue by olechnwin** [phasegenomics/FALCON-Phase#64]
Hi,

I am trying to install falcon-phase but when I do 
`git clone --recursive https://github.com/phasegenomics/FALCON-Phase.git ; cd FALCON-Phase/src ; make`

I got the following error:
```
cd ../htslib && make
make[1]: Entering directory `/gpfs0/home/cxt050/opt/FALCON-Phase/htslib'
gcc -g -Wall -O2 -I.  -c -o bgzf.o bgzf.c
In file included from bgzf.c:39:0:
htslib/bgzf.h:35:10: fatal error: zlib.h: No such file or directory
 #include <zlib.h>
          ^~~~~~~~
compilation terminated.
make[1]: *** [bgzf.o] Error 1
make[1]: Leaving directory `/gpfs0/home/cxt050/opt/FALCON-Phase/htslib'
make: *** [../htslib/libhts.a] Error 2

```

I have installed all the dependencies listed. Please let know what I need to do. Thank you!

**Comment by shawnpg** [comment]
Hi,

Have you tried installing via PacBio's conda environment https://github.com/PacificBiosciences/pbbioconda? Official support for FALCON-Phase has moved there and it's usually much easier to get the environment up and running. If that won't work for you, let us know and we'll investigate.

Thanks!

**Comment by olechnwin** [comment]
@shawnpg thank you so much for your quick reply. I haven't tried PacBio's conda environment.
I'll try it and post back. Thanks!

**Comment by olechnwin** [comment]
@shawnpg, Sorry. I don't really understand how to install FALCON-Phase using PacBio's conda environment. I don't see FALCON-Phase on the list of packages. Do I install pb-assembly?

**Comment by shawnpg** [comment]
Hi,

It's included in the pb-falcon package. Sorry I didn't mention it. It's buried in the release notes (https://github.com/PacificBiosciences/pbbioconda/wiki/Release-notes), but it's there as of January.

Thanks!

**Comment by olechnwin** [comment]
I've installed the entire pb-assembly which also includes falcon-phase package.
How do I run it the test dataset? Do I still follow the readme in this page? Do I still clone the repo and use the test data, config.sh and config.json?
If I do need to follow the readme, then I need to install snakemake as it is not installed with pb-assembly.

Thanks!

**Comment by zeeev** [comment]
Hi @olechnwin, 

Currently there is no documentation for running FALCON-Phase with pb-assembly. I'm still recommending that people use this repository since it has been extensively tested and continually supported. 

The original error you reported is that a system library is missing. The zlib library is required by htslib. If you have samtools installed on your system chances are that you've also installed htslib. FALCON-phase uses htslib to read the bam files.

Sorry for the confusion. 

--Zev

**Comment by olechnwin** [comment]
Ok. I was able to fix the library error by loading gcc 7.2.0 library before compiling.

I ran the test dataset and have the following warning:
```
Warning: error in phasing, did you run enough iterations ('iter' >= 10e6)? 000000F_001:0-26421 and 000000F:10327801-10353990 'phased.txt' contains '-nan'
Warning: error in phasing, did you run enough iterations ('iter' >= 10e6)? 000000F_003:0-35964 and 000000F:21633239-21669154 'phased.txt' contains '-nan'
```

Is it supposed to throw this warning? What does it mean?

**Comment by zeeev** [comment]
Hi @olechnwin,

We updated the default number of permutations required. I didn't update the sample config. If you increase the number of iterations you should be good.
